fusor-ember-cli/app/controllers/configure-environment.js

```diff
@@ -4,8 +4,8 @@ import NeedsDeploymentMixin from "../mixins/needs-deployment-mixin";
 
 export default Ember.Controller.extend(ConfigureEnvironmentMixin, NeedsDeploymentMixin, {
 
-  satelliteTabRouteName: Ember.computed.alias("deploymentController.model.satelliteTabRouteName"),
-  organizationTabRouteName: Ember.computed.alias("deploymentController.model.organizationTabRouteName"),
+  satelliteTabRouteName: Ember.computed.alias("deploymentController.satelliteTabRouteName"),
+  organizationTabRouteName: Ember.computed.alias("deploymentController.organizationTabRouteName"),
```

Comment:
this is a separate bug fix where the back button wasn't working on the Lifecycle Environment page.